user-data/main.tf

```diff
@@ -1,3 +1,13 @@
+data "template_file" "prometheus_service_cloudinit" {
+  template = "${file("${path.module}/templates/prometheus.service.tpl")}"
+
+  vars {
+    prometheus_service = "${indent(4,file("${path.module}/templates/prometheus-upstart.conf"))}"
+    service_type_path  = "/etc/init.d/prometheus"
```

Comment:
Could we call it `node-exporter` instead of `prometheus`? It's a bit confusing otherwise, as it's not the prometheus process that's running.